pom.xml

```diff
@@ -79,6 +79,25 @@
         </overridesfile>
     </properties>
 
+    <repositories>
+        <repository>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <id>kogito</id>
+            <name>kogito</name>
+            <url>https://repository.jboss.org/nexus/content/groups/kogito-public/</url>
+        </repository>
+        <repository>
+            <snapshots>
+            <enabled>false</enabled>
+            </snapshots>
+            <id>central</id>
+            <name>Maven Central</name>
+            <url>https://repo.maven.apache.org/maven2</url>
+        </repository>
```

Comment:
Is this really needed?

Reply:
@evacchi ? :D

Reply:
If they have been properly released to Maven Central, this is not necessary.

Reply:
they are not yet released to Maven Central, That is why we have also https://github.com/quarkusio/quarkus-platform/pull/239/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R83-R90

Reply:
before releasing we check that the platform PR is ok :)

Reply:
@gsmet @geoand Could we at least launch the check ?
anyway this will be removed before merging the PR ...

Reply:
removed

Reply:
ah yes sorry about not launching the check, not used to this new GH feature. As far as I can see CI is green now.